src/components/Article.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Article = ({
+  title,
+  text,
+  date,
+}) => (
```

Comment:
No need to add extra lines, code is short enough.

```suggestion
const Article = ({ title, text, date }) => (
```